tests/actions.go

```diff
@@ -49,14 +49,10 @@ func NewOperatorActions(cli versioned.Interface, kubeCli kubernetes.Interface) O
 
 const (
 	DefaultPollTimeout  time.Duration = 10 * time.Minute
-	DefaultPollInterval time.Duration = 1 * time.Minute
+	DefaultPollInterval time.Duration = 10 * time.Second
```

Comment:
Why change this to 10 seconds, it will be very noisy

Reply:
xiaojing and I need to check the log and scheduledbackup-cronjob is called job every minute